### ISSUE TITLE
Radius too small from `boundingCircle()`

### ISSUE THREAD

**Issue by EastEriq** [EranOfek/AstroPack#213]
I have a catalog with these properties:
```
>> Cat=AI(1).CatData;
>> Cat.ColNames{52:53}

ans =

    'RA'


ans =

    'Dec'

>> Cat.ColUnits{52:53}

ans =

    'deg'


ans =

    'deg'

>> [min(Cat.Catalog(:,52)),max(Cat.Catalog(:,52))]

ans =

       140.81       141.45

>> [min(Cat.Catalog(:,53)),max(Cat.Catalog(:,53))]

ans =

      -25.336       -24.76
```

On that, `boundingCircle()` gives me:

```
>> [RA, Dec, CircleRadius] = boundingCircle(Cat, 'OutUnits','deg', 'CooType','sphere')

RA =

       141.22


Dec =

      -24.761


CircleRadius =

   0.00013031
```

The resulting `RA` makes sense (midway between min and max), but `Dec` and `CircleRadius` not. I'd expect `CircleRadius` to be of the order of 0.5 degrees. It comes out instead 0.46911 arcsecs.

**Comment by EastEriq** [comment]
https://github.com/EranOfek/AstroPack/blob/7d710349c68ee580c100f2cac7ba96b2bd2c2aae/matlab/astro/%2Bcelestial/%2Bcoo/boundingCircle.m#L52
Fuck you. #156

**Comment by EastEriq** [comment]
Predictably, `celestial.coo.sphere_dist_fast(...,false)` restores a plausible result.

**Comment by EranOfek** [comment]
please send me the entire Cat object so I can debug it - please send to my gmail (WIS mail doesnt forward mat files...)

**Comment by EastEriq** [comment]
You didn't read, it is not the catalog, it is #156

**Comment by EastEriq** [comment]
fixed by commit 9ce248d62f.
